packages/core/src/libraries/user.ts

```diff
@@ -47,13 +47,60 @@ export const verifyUserPassword = async (user: Nullable<User>, password: string)
   return user;
 };
 
+const converBindMfaToMfaVerification = (bindMfa: BindMfa): MfaVerification => {
```

Comment:
why do we need to convert here? better to have a brief description in a **TSDoc** comment block

Reply:
updated